deploy/deploy.yaml

```diff
@@ -103,7 +103,7 @@
                      "./event_exporter"
                   ],
                   "env": [ ],
-                  "image": "cargo.caicloud.io/sysinfra/event-exporter",
+                  "image": "caicloud/event-exporter:v0.2.0",
```

Comment:
the registry used here should be consistent to the Makefile, which is updated in #32

Reply:
This deployment is to help users to run this exporter. The images need to be pushed into Docker hub after released.
The latest released version is [v0.2.0](https://hub.docker.com/layers/caicloud/event-exporter/v0.2.0).